CODEOWNERS

```diff
@@ -0,0 +1,53 @@
+# Remember that the last applicable rule for any given file is the only one
+# that applies (if I am reading the docs correctly).
+
+# Default rule: @diyessi gets everything.
+*                                   @diyessi
+
+/cmake/                             @rkimballn1 @silee2
+
+/.ci/                               @crlishka
+/contrib/docker/                    @crlishka
+/.travis.yml                        @postrational
+
+/.clang-format                      @rkimballn1
+/.gitattributes                     @rkimballn1
+/.gitignore                         @rkimballn1
+/VERSION.in                         @rkimballn1 @silee2
+
+/maint/                             @rkimballn1
+
+/doc/                               @indie
+/README.md                          @indie
+/changes.md                         @indie @diyessi
+/INSTALL.md                         @indie
+/CONTRIB.md                         @indie
+
+/licenses/                          @diyessi @kimjanik @crlishka
+/LICENSE                            @diyessi @kimjanik @crlishka
+
+/python/                            @postrational
+
+/src/resource/                      @rkimballn1
+/src/tools/                         @rkimballn1
+/src/ngraph/autodiff/               @diyessi
+/src/ngraph/builder/                @mbrookhart
+/src/ngraph/codegen/                @rkimballn1
+/src/ngraph/frontend/               @postrational
+/src/ngraph/op/                     @diyessi @aprocter
+/src/ngraph/pass/                   @Krovatkin
+/src/ngraph/pattern/                @Krovatkin
+/src/ngraph/runtime/                @rkimballn1 @jbobba
+/src/ngraph/runtime/cpu/            @jbobba
+/src/ngraph/runtime/gpu/            @rkimballn1
+/src/ngraph/runtime/intelgpu/       @shssf
+/src/ngraph/runtime/interpreter/    @rkimballn1
+/src/ngraph/runtime/reference/      @aprocter
+/src/ngraph/type/                   @diyessi
+/src/ngraph/serializer.*pp          @rkimballn1
+
+# Putting this last so it's not overridden by directory rules
+CMakeLists.txt                      @rkimballn1 @silee2
+
+# Putting this last to make sure nothing else overrides.
+/CODEOWNERS                         @diyessi
```

Comment:
Why not Adam Proctor for this ?

Reply:
Talked to @diyessi and we're thinking it makes the most sense to have him on it. He can always tag me if there's some issue about the formatting of the file or something.